client/src/utils/string.ts

```diff
@@ -0,0 +1,59 @@
+/**
+ * Capitalise a string
+ *
+ * @param string String to capitalise
+ * @returns Capitalised string
+ */
+export function capitalise(string: string) {
```

Comment:
Can you quickly add return types to each of the functions? I know that typescript can infer the  resultant type, but it's better for us to be explicit so that typescript doesn't automatically add on a different type.

See https://stackoverflow.com/questions/53661635/when-is-it-necessary-to-declare-return-function-type-in-typescript for more detail.